packages/server/src/publicProsopoPair.ts

```diff
@@ -0,0 +1,16 @@
+import { KeypairType } from '@polkadot/util-crypto/types'
+import { getPair } from '@prosopo/common'
+
+export type PublicAccountNetwork = 'rococo'
+
+export const getPublicProsopoPair = async (network: PublicAccountNetwork) => {
+    const accountNetworkMap = {
+        rococo: 'open poet tennis test nice immune scorpion flag cement surge effort canal', // Prosopo rococo shared public account
+    }
+
+    const pairType = 'sr25519' as KeypairType
+    const ss58Format = 42
+    const prosopoSharedAccountSecret = accountNetworkMap[network]
```

Comment:
`prosopoSharedAccountSecret` must be `string | undefined`, no? Think you need to use the `get` fn from utils

Reply:
Just comes up as string in my IDE - typescript actually being clever for once?

Reply:
ah no, my bad. It's because you've set the type to `PublicAccountNetwork` rather than string. All good :)